minkindr/include/kindr/minimal/transform-2d.h

```diff
@@ -99,6 +99,10 @@ using Position2D = Position2DTemplate<double>;
 using Rotation2D = Rotation2DTemplate<double>;
 using Transformation2D = Transformation2DTemplate<double>;
 
+template<typename Scalar>
+std::ostream & operator<<(std::ostream & out,
```

Comment:
can you include the `<ostream>` header for that?

Reply:
Other files don't have it. For example [this](https://github.com/ethz-asl/minkindr/blob/master/minkindr/include/kindr/minimal/angle-axis.h) and [this](https://github.com/ethz-asl/minkindr/blob/master/minkindr/include/kindr/minimal/quat-transformation.h).

Reply:
but that doesn't mean that it's not needed :wink:

Reply:
touché. I added it to all needed files.